docs/readthedocs/source/doc/Chronos/Overview/speed_up.md

```diff
@@ -127,15 +127,15 @@ f.load(checkpoint_file="fp32.model"
 ```
 Please refer to [Forecaster API Docs](../../PythonAPI/Chronos/forecasters.html) for details.
 
-#### **3.2 `TSPipeline` Inference Acceleration**
+#### 3.2 `TSPipeline` Inference Acceleration
 Basically same to [`Forecaster`](#31-forecaster-inference-acceleration)
-##### **3.1.1 Default Acceleration**
+##### 3.1.1 Default Acceleration
```

Comment:
maybe 3.1.1 --> 3.2.1 ? Line 134, 138 also.

Reply:
Thank you! Have fixed it :D